litex/soc/cores/clock/lattice_ecp5.py

```diff
@@ -12,19 +13,21 @@
 # Lattice / ECP5 -----------------------------------------------------------------------------------
 
 class ECP5PLL(Module):
-    nclkouts_max    = 3
+    nclkouts_max    = 4
     clki_div_range  = (1, 128+1)
     clkfb_div_range = (1, 128+1)
     clko_div_range  = (1, 128+1)
     clki_freq_range = (    8e6,  400e6)
     clko_freq_range = (3.125e6,  400e6)
     vco_freq_range  = (  400e6,  800e6)
+    pfd_freq_range  = (   10e6,  400e6)
```

Comment:
I believe the `clki_freq_range` above is supposed to be the same as this. They should probably be combined.

Reply:
There are actually two separate specs for clki/clkfb and pfd:

![Screenshot_20210707_090816](https://user-images.githubusercontent.com/3952613/124773818-e9c71880-df02-11eb-9b76-eabbe64d3a1e.png)

The footnote says:
>Period jitter and cycle-to-cycle jitter numbers are guaranteed for fPFD > 10 MHz. For fPFD < 10 MHz, the jitter numbers may not be met in certain conditions

This is a "soft" spec. It sounds like it will work without meeting it, however Diamond refuses to solve PLL configs where this spec is violated. I can be persuaded to leave it out.

Reply:
Ah, fair. Still seems a bit unintuitive that the minimum is higher for f_PFD than f_IN, considering the former is derived from the latter through a divisor.

I can also confirm that for my sample size of one, it works at 5 MHz in practice. :)